frontend/src/metabase/forms/components/FormSubmitButton/FormSubmitButton.tsx

```diff
@@ -55,7 +63,7 @@ type SubmitButtonColorOpts = Pick<FormSubmitButtonProps, "color">;
 const getSubmitButtonColor = (
   status: FormStatus | undefined,
   { color }: SubmitButtonColorOpts,
-) => {
+): MantineColor | undefined => {
```

Comment:
Why do we need to type color here? If we don't have this change I assume we also don't need to introduce the color type `MantineColor`.

After removing the type here, `yarn type-check` still passed locally.

Reply:
there was a good reason I had this here, but now I can't remember. I'll remove it